### PR TITLE
ALBS-220 Add modules to release preview with ability to choose repo

### PR DESCRIPTION
**Pull request by maccelf**
Release preview allows to choose repo for every package.
The same should be done for module metadata. Under packages should be form with modules (if they are in release).
NAME:STREAM:VERSION:CONTEXT and repo to release. By default repo should be chosen automatically by Beholder, but it’s important to see and change what will be released and where.

### CODE REVIEW

**Review by Korulag — APPROVED** [review]

File: src/components/PackageLocationSelectionForm.vue [hunk 1]
```diff
@@ -98,6 +105,53 @@
                     </q-td>
                 </q-tr>
             </template>
+            <template v-if="modules && modules.length" v-slot:bottom-row>
+                <q-tr>
+                    <q-th colspan="100%" class="text-left">
+                        Modules
+                    </q-th>
+                </q-tr>
+                <q-tr v-for="modul in modules" :key="modul.arch">
```
Comment (anfimovdm) [inline src/components/PackageLocationSelectionForm.vue:114]: typo in modul, should be module

File: src/components/PackageLocationSelectionForm.vue [hunk 2]
```diff
@@ -145,6 +200,9 @@ export default defineComponent({
         tableFullScreen(props){
             props.toggleFullscreen()
         },
+        nsvca (modul){
```
Comment (anfimovdm) [inline src/components/PackageLocationSelectionForm.vue:203]: same above, i'm think we already have same func in other place, can we reuse it?

**Review by soksanichenko — APPROVED** [review]

File: src/components/PackageLocationSelectionForm.vue (hunk 2 above)
Comment (anfimovdm) [reply src/components/PackageLocationSelectionForm.vue]: pls define `nsvca` func in one place and import this func for `Build`, `BuildFeedItem`, `PackageLocationSelectionForm`

Comment (maccelf) [reply src/components/PackageLocationSelectionForm.vue]: Done

**Review by anfimovdm — APPROVED** [review]

**Pull request merged**